catkit2/services/thorlabs_cube_motor_kinesis/thorlabs_cube_motor_kinesis.py

```diff
@@ -88,7 +88,11 @@ def open(self):
         """
         # Build the device list and verify that the device is connected.
         if self.lib.TLI_BuildDeviceList() == 0:
-            serial_number_list_size = 35  # 4 * 8 (SN size) + 3 for the commas between the serial numbers
+            # The serial_number_list_size is the size of a list that holds the cubes' serial numbers.

```

Comment:
Let's add: "...of one motor type." to the end of this sentence, since serial numbers are queried by type.